controllers/apply/lib/field-types/day-month.js

```diff
@@ -0,0 +1,50 @@
+'use strict';
+const moment = require('moment');
+const Joi = require('../joi-extensions');
+
+const Field = require('./field');
+
+class DayMonthField extends Field {
+    getType() {
+        return 'day-month';
+    }
+
+    defaultSchema() {
+        return Joi.dayMonth().required();
+    }
+
+    defaultMessages() {
+        return [
+            {
+                type: 'base',
+                message: this.localise({
```

Comment:
Because these message don't reference anything to do with the specific field (e.g. accounting year end) we can included them as default messages